loleaflet/src/core/Socket.js

```diff
@@ -268,6 +268,30 @@ L.Socket = L.Class.extend({
 			     'background:#ddf;color:black', color, 'color:black');
 	},
 
+	// The problem: if we process one websocket message at a time, the
```

Comment:
Right. This wasn't such a big issue, but I suppose increasingly it is, as the efficiency of round-trips improved.